code/datums/mutations/body.dm

```diff
@@ -96,7 +96,7 @@
 
 //Tourettes causes you to randomly stand in place and shout.
 /datum/mutation/human/tourettes
-	name = "Tourettes Syndrome"
+	name = "Tourette's Syndrome"
```

Comment:
Arguably tourette syndrome or tourette's though muh socialised medicine does call it this online.